hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetCreator.cpp

```diff
@@ -336,22 +331,16 @@ void ChangesetWriter::_handleUnstreamableConvertOpsInMemory(const QString& input
   {
     // Just load the first map, but as unknown2 to end up with a changeset made up of just this
     // input.
-    if (ConfigUtils::boundsOptionEnabled())
-    {
-      conf().set(
-        ConfigOptions::getConvertBoundingBoxKeepEntireFeaturesCrossingBoundsKey(),
-        ConfigOptions().getChangesetReferenceKeepEntireFeaturesCrossingBounds());
-      conf().set(
-        ConfigOptions::getConvertBoundingBoxKeepOnlyFeaturesInsideBoundsKey(),
-        ConfigOptions().getChangesetReferenceKeepOnlyFeaturesInsideBounds());
-    }
     IoUtils::loadMap(fullMap, input1, true, Status::Unknown2);
```

Comment:
These changeset specific bounds option changes were only needed when I first pre-prototyped the replacement changeset generation as a set of calls to other hoot commands. No longer needed now that everything is done in a separate command.